src/Render.cc

```diff
@@ -16,11 +16,15 @@ using namespace snowcrash;
 namespace drafter {
 
     // JSON content-type regex
-    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json";
+    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json[[:blank:]]*(;.*|$)";
+    const char* const JSONSchemaRegex = "^[[:blank:]]*application/schema\\+json[[:blank:]]*(;.*|$)";
```

Comment:
`asdf application/schema+json` would fail and substring wouldn't match. Because there is an `^` at the beginning of the regex which means content type should start with `application/schema+json` (blank space excluded) and either end there or possibly have any parameters after a semicolon.